### PR TITLE
Upload presentations and still have default.pdf

### LINKED ISSUE #14828: when preloading a presentation from the greenlight interface in bbb, the presentation default.pdf is not displayed
**Describe the bug**
when preloading a presentation from the greenlight interface in bbb, the presentation default.pdf is not displayed

**To Reproduce**
1. Go to greenlight interface
2. Click on 'add presentatoin' in the greenlight interface
3. Click on 'Start'
4. We see the presentation that was uploaded to greenlight, but does not see the presentation default.pdf

**Expected behavior**
Two presentations are available: 1st one that I uploaded, 2nd default.pdf.

**Actual behavior**
there is only one presentation available, which I downloaded from the greenlight interface. presentation default.pdf is not available for display

**Screenshots**
image

image

**BBB version:**
`$ sudo bbb-conf --check | grep BigBlueButton`
BigBlueButton Server 2.4.6 (3059)
Greenlight. release-2.12.3

**Desktop:**
 - OS: [Windows, Mac]
 - Browser [Chrome, Safari]
 - Device: [hp elitebook g5, MacBook air (m1)]

**Smartphone:**
 - Device: [iPhone11]
 - OS: [iOS 15.3.1]
 - Browser [stock browser, Safari]

**Additional context**
**Question:** should the default.pdf presentation be available in the bbb interface when preloading the presentation from the greenlight interface, or should the default.pdf presentation be unavailable for display if we loaded our presentation from the greenlight interface?

### PR DESCRIPTION
**Pull request by GuiLeme**
### What does this PR do?

It changes the flow to preupload some documents: now with the parameter `preUploadedPresentationOverrideDefault=false`, it is possible to have the documents preuploaded along with the `default.pdf` too. 

### Closes Issue(s)

Closes #14828

### CODE REVIEW

File: bigbluebutton-web/grails-app/conf/bigbluebutton.properties [hunk 1]
```diff
@@ -363,6 +363,9 @@ beans.presentationService.testPresentationName=appkonference
 beans.presentationService.testUploadedPresentation=appkonference.txt
 # Default Uploaded presentation file
 beans.presentationService.defaultUploadedPresentation=${bigbluebutton.web.serverURL}/default.pdf
+# Decides weather to send the default.pdf document along with the other presentations in create endpoint (false)
+# or not (true / default)
```
Comment (gustavotrott) [inline bigbluebutton-web/grails-app/conf/bigbluebutton.properties:367]:
```suggestion
# Discard default presentation (default.pdf) when Pre-upload Slides are sent within the create call (default true)
```

**Review by gustavotrott — CHANGES_REQUESTED** [review]

**Comment by gustavotrott** [comment]
I was expecting `Default.pdf` to be the default (current). Once I didn't assigned any other as `current`.

https://user-images.githubusercontent.com/5660191/166939630-ed3ec2e4-002c-4292-af00-2088ff51878c.mp4

**Comment by GuiLeme** [comment]
> I was expecting `Default.pdf` to be the default (current). Once I didn't assigned any other as `current`.

Yeah, I thought that if any is assigned to be current, the default flow would be executed, which is the first document in the payload batch will be the current, but I can quickly change it, no worries. So I will fix it and then make another commit, ok?

**Comment by GuiLeme** [comment]
> I was expecting `Default.pdf` to be the default (current). Once I didn't assigned any other as `current`.

Already implemented, thanks for the suggestion!

**Review by gustavotrott — APPROVED** [review]
LGTM!

**Comment by antobinary** [comment]
Hi @GuiLeme Could you please confirm it works as well when using GreenLight? ( #14828 was reported with GreenLight)

**Review by gustavotrott — APPROVED** [review]
Great work @GuiLeme! Working very well!

**Pull request merged**